pywebostv/controls.py

```diff
@@ -176,6 +181,20 @@ class MediaControl(WebOSControlBase):
         "stop": {"uri": "ssap://media.controls/stop"},
         "rewind": {"uri": "ssap://media.controls/rewind"},
         "fast_forward": {"uri": "ssap://media.controls/fastForward"},
+        "get_audio_output": {
+            "uri": "ssap://audio/getStatus",
+            "validation": standard_validation,
+            "subscription": True,
```

Comment:
```suggestion
            "subscription": True,
            "return": AudioOutputSource
```

That should convert the incoming response from LG TV Server to an AudioOutputSource instance. This is untested, though :)

If this request returns a more complex object, you'd have to do something like ApplicationControl -> list_apps -> "return" (line number 248 below).

Reply:
Also, can you please dump the entire response as you get from the TV? There might be some other interesting fields..

Reply:
its just {'soundOutput': 'external_speaker'}

Reply:
Ahh, thanks so much. Just a few things to note in that case:

You might want to remove `"standard_validation"` then (the entire key+value). I am guessing it would fail because of this line: https://github.com/supersaiyanmode/PyWebOSTV/blob/master/pywebostv/controls.py#L117?

Reply:
What is the response if you print out the object here: https://github.com/supersaiyanmode/PyWebOSTV/blob/master/pywebostv/connection.py#L194 ?

Reply:
**received_message() output:** 
{"type":"response","id":"5a1a03b3-03d1-4264-b83e-cf89be987db0","payload":{"returnValue":true,"soundOutput":"external_speaker"}}

**Output with standard_validation:** 
{'soundOutput': 'external_speaker'}

**Output without standard_validation:** 
{'returnValue': True, 'soundOutput': 'external_speaker'}

So I guess it works, right?

Reply:
Yes, it works with standard_validation :). Let's not remove standard_validation then